src/main/kotlin/com/openlattice/chronicle/study/StudyApi.kt

```diff
@@ -43,5 +40,15 @@ interface StudyApi {
     @GET(BASE + STUDY_ID_PATH)
     fun getStudy(@Path(STUDY_ID) studyId: UUID): Study
 
-
+    /**
+     * Updates an existing study based on id
+     * @param studyId The id of the study to retrieve.
```

Comment:
`to retrieve` ... looks like copy/paste bug?

```suggestion
     * @param studyId The id of the study to update.
```